<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogDeleteAccount</class>
 <widget class="QDialog" name="dialogDeleteAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete Account</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QComboBox" name="comboChooseAccount">
       <item>
        <property name="text">
         <string>NoAccountSaved</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="1">
          <widget class="QPushButton" name="buttonDeleteAccount">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="buttonCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choose an account to delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
